SSS_task/ViewControllers/CategoryControllers/PLP_VC/ViewController/ProductDetailCollectionCell/ProductDetailCollectionCell.swift

```diff
@@ -100,7 +100,7 @@ class ProductDetailCollectionCell: UICollectionViewCell {
                 let json = try JSONSerialization.jsonObject(with: data, options: .mutableContainers)
                 print(json)
                 let wishlistData = try? JSONDecoder().decode(AddWishlistModel.self, from: data)
```

Comment:
Where is try catch for JsonDescoder?

Reply:
removed it, used it only to check if data is receiving or not